README.md

```diff
@@ -4,10 +4,38 @@ This project was generated using [Nx](https://nx.dev).
 
 ## Quick Start
 
+### Setup
+
 - Optional: `npm i -g nx`
   - This will allow you to run `nx` commands without `npx`. If you do not want to install `nx` globally, you can run `npx nx` instead of `nx`.
 - Install dependencies: `npm install`
-- Start Expo app: `nx start mobile-app`
+
+### Running the Mobile App
+
+You'll need two terminal windows:
+
+1. Start the Expo development server:
+
+   - `nx start mobile-app`
+
+2. In a second terminal, build and run the iOS app:
+   - `nx ios mobile-app`
```

Comment:
I've just been running `nx ios mobile-app` and it works lol

Reply:
oh wut

Reply:
yup, you're right